src/Events/Controllers/AppEventsController.cs

```diff
@@ -83,7 +83,8 @@ public async Task<ActionResult<string>> Post([FromBody] CloudEventRequestModel c
 
             try
             {
-                string cloudEventId = await _eventsService.StoreCloudEvent(_mapper.Map<CloudEvent>(cloudEvent));
+                string cloudEventId = await _eventsService.SaveAndPushToInboundQueue(_mapper.Map<CloudEvent>(cloudEvent));
+                _logger.LogInformation("Cloud Event successfully stored with id: {cloudEventId}", cloudEventId);
```

Comment:
Do we need these log messages or we're they just used for development? I guess an exception will be thrown if it isn't successful.

Reply:
I would think the same :) I would suggest we delete the logging of info